environments/64-test/public.yml

```diff
@@ -0,0 +1,177 @@
+
+couch_dbs:
+  default:
+    host: "{{ groups.couchdb2.0 }}"
+    port: "{{ couchdb2_proxy_port }}"
+    name: commcarehq
+    username: "{{ localsettings_private.COUCH_USERNAME }}"
+    password: "{{ localsettings_private.COUCH_PASSWORD }}"
+    is_https: False
+  users:
+    host: "{{ groups.couchdb2.0 }}"
+    port: "{{ couchdb2_proxy_port }}"
+    name: commcarehq__users
+    username: "{{ localsettings_private.COUCH_USERNAME }}"
+    password: "{{ localsettings_private.COUCH_PASSWORD }}"
+    is_https: False
+
+couchdb_cluster_settings:
+  q: 8
+  r: 1
+  w: 1
+  n: 1
+
+TOUCHFORMS_API_USER: 'touchforms@example.com'
+
+localsettings:
+  ALLOWED_HOSTS:
+    - localhost
+    - 127.0.0.1
+    - "{{ SITE_HOST }}"
+  CELERY_PERIODIC_QUEUE: 'celery_periodic'
+  CELERY_RESULT_BACKEND: 'database'
+  DEPLOY_MACHINE_NAME: "{{ inventory_hostname }}"
+  ELASTICSEARCH_HOST: "{{ groups.elasticsearch.0 }}"
+  ELASTICSEARCH_PORT: '9200'
+
+  EMAIL_SMTP_HOST: email-smtp.us-east-1.amazonaws.com
+  EMAIL_SMTP_PORT: 587
+  EMAIL_USE_TLS: yes
+  FULLSTORY_ID:
+  HQ_INSTANCE: 'dev'
+  J2ME_ADDRESS: ''
+  KAFKA_URL: "{{ groups.kafka.0 }}:9092"
+  LOCAL_PILLOWS:
+    custom:
+      - name: 'kafka-ucr-main-custom-1'
+        class: 'corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow'
+        instance: 'corehq.apps.userreports.pillow.get_kafka_ucr_static_pillow'
+        params:
+          ucr_division: '09'
+      - name: 'kafka-ucr-main-custom-2'
+        class: 'corehq.apps.userreports.pillow.ConfigurableReportKafkaPillow'
+        instance: 'corehq.apps.userreports.pillow.get_kafka_ucr_static_pillow'
+        params:
+          ucr_division: 'af'
+  REDIS_DB: '0'
+  REDIS_HOST: "{{ groups.redis.0 }}"
+  REDIS_PORT: '6379'
+  STATIC_CDN: ""
+  FORMPLAYER_URL: "http://{{ groups.formplayer.0 }}:8181"
+
+root_email: droberts+root@dimagi.com
+server_email: droberts+commcarehq-noreply@dimagi.com
+default_from_email: droberts+commcarehq-noreply@dimagi.com
+support_email: droberts+support@dimagi.com
+probono_support_email: droberts+pro-bono@dimagi.com
+cchq_bug_report_email: droberts+commcarehq-bug-reports@dimagi.com
+accounts_email: droberts+accounts@dimagi.com
+data_email: droberts+datatree@dimagi.com
+subscription_change_email: droberts+accounts+subchange@dimagi.com
+internal_subscription_change_email: droberts+accounts+subchange+internal@dimagi.com
+billing_email: droberts+billing-comm@dimagi.com
+invoicing_contact_email: droberts+billing-support@dimagi.com
+growth_email: droberts+growth@dimagi.com
+master_list_email: droberts+master-list@dimagi.com
+report_builder_add_on_email: droberts+sales@dimagi.com
+eula_change_email: droberts+eula-notifications@dimagi.com
+contact_email: droberts+info@dimagi.com
+soft_assert_email: droberts+commcarehq-ops+soft_asserts@dimagi.com
+daily_deploy_email: null
+
+commcare_cloud_root_user: ubuntu
+commcare_cloud_strict_host_key_checking: no
+commcare_cloud_pem: /Users/droberts/.ssh/commcarehq_testing.pem
```

Comment:
is this right?

Reply:
Yeah, well, this is just what I used to bootstrap it; no one else needs this file to work on this env. Since it doesn't make much of a difference I think I'm just going to leave it